### PR TITLE
feat/create-treasury-wallet

### PR DESCRIPTION
**Pull request by 0xdevcollins**
working progress

### CODE REVIEW

File: lib/wallet/generateWallet.ts
Comment (jonesjBSV) [inline lib/wallet/generateWallet.ts:1]: Looks good +1

File: app/api/wallet/balance/route.ts
Comment (jonesjBSV) [inline app/api/wallet/balance/route.ts:1]: Don't hardcode WIFs/keys -- either generate a new one with each transaction (you can use the type42 style derivations built-in to the @bsv/sdk to do this)

or

Take them as input. never use the same key twice (testing being the exception)

Comment (0xdevcollins) [reply app/api/wallet/balance/route.ts:1]: Noted sir

**Comment by mo-jaber** [comment]
Looking good @0xdevcollins. Could you just run the following database schema migration scripts for any tables and columns you may have created ? 

https://github.com/bitcoin-sv/bsv-faucet?tab=readme-ov-file#database-schema-management

**Comment by mo-jaber** [comment]
@0xdevcollins could you have another look to resolve the conflicts and also run the https://github.com/bitcoin-sv/bsv-faucet?tab=readme-ov-file#database-schema-management

**Pull request merged**